docs/modules/mssqlfg.md

```diff
@@ -0,0 +1,300 @@
+  -H 'content-type: application/json' \
+  -H 'x-broker-api-version: 2.13' \
+  -d '{
+    "service_id" : "a7454e0e-be2c-46ac-b55f-8c4278117525",
+    "plan_id" : "24f0f42e-1ab3-474e-a5ca-b943b2c48eee",
+    "parameters" : {
+      "primaryResourceGroup":"osba",
+      "primaryServer":"osbasql1",
+      "primaryLocation":"eastus",
+      "primaryAdministratorLogin":"username1",
+      "primaryAdministratorLoginPassword":"password1",
+      "secondaryResourceGroup":"osba",
+      "secondaryServer":"osbasql2",
+      "secondaryLocation":"westus",
+      "secondaryAdministratorLogin":"username2",
+      "secondaryAdministratorLoginPassword":"password2",
+      "alias":"serverpair"
+    }
+}'
+```
+
+### Service: azure-sql-12-0-dr-database-pair
+
+| Plan Name | Description |
+|-----------|-------------|
+| `basic` | Basic Tier, 5 DTUs, 2GB, 7 days point-in-time restore |
+| `standard` | Standard Tier, Up to 3000 DTUs, with 250GB storage, 35 days point-in-time restore |
+| `premium` | Premium Tier, Up to 4000 DTUs, with 500GB storage, 35 days point-in-time restore |
+| `general-purpose` | General Purpose Tier, Up to 80 vCores, Up to 440 GB Memory, Up to 1 TB storage, 7 days point-in-time restore |
+| `business-critical` | Business Critical Tier, Up to 80 vCores, Up to 440 GB Memory, Up to 1 TB storage, Local SSD, 7 days point-in-time restore. Offers highest resilience to failures using several isolated replicas |
+
+#### Behaviors
+
+##### Provision
+
+Provisions a pair of new databases upon both the primary server and the secondary server. And these two databases make up a failover group. If the DBMS pair does not yet exist, provision of the database will be deferred until the DBMS has been provisioned.
+
+###### Provisioning Parameters
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `parentAlias` | `string` | Specifies the alias of the DBMS upon which the database should be provisioned. | Y | |
+| `database` | `string` | Specifies the name of the databases. | Y | |
+| `failoverGroup` | `string` | Specifies the name of the failover group. | Y | |
+
+Additional Provision Parameters for : standard plan
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `dtus` | `integer` | Specifies Database transaction units, which represent a bundled measure of compute, storage, and IO resources. Valid values are 10, 20, 50, 100, 200, 400, 800, 1600, 3000 | N | 10 |
+
+
+Additional Provision Parameters for : premium plan
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `dtus` | `integer` | Specifies Database transaction units, which represent a bundled measure of compute, storage, and IO resources. Valid values are 125, 250, 500, 1000, 1750, 1000 | N | 125 |
```

Comment:
I guess it should be "1750, 2000"?

Reply:
It is 4000 from the code. Sorry that I didn't check the details and just copied them from mssql... Will fix.